### ISSUE TITLE
Add ability to perform hyperparameter search over loss function weights

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/Arbiter#145]
Currently, this isn't really possible...

**Comment by AlexDBlack** [comment]
https://github.com/deeplearning4j/deeplearning4j/issues/5173
